src/test/java/com/wikia/webdriver/testcases/desktop/forumtests/ForumEditModeTests.java

```diff
@@ -60,13 +60,16 @@ public void adminUserCanDeleteBoard() {
     ForumBoardPage forumBoard = new ForumBoardPage();
     first = forumBoard.createNew(User.USER_ADMIN_FORUM);
     second = forumBoard.createNew(User.USER_ADMIN_FORUM);
-
     ForumPage forumMainPage = new ForumPage();
     forumMainPage.openForumMainPage(wikiURL);
     ForumManageBoardsPageObject manageForum = forumMainPage.clickManageBoardsButton();
+    description = PageContent.FORUM_DESCRIPTION_PREFIX + manageForum.getTimeStamp();
+
     manageForum.verifyForumExists(first, wikiURL);
-    manageForum.deleteForum(first, second);
-    manageForum.verifyForumNotExists(first, wikiURL);
+    manageForum.verifyForumExists(second, wikiURL);
```

Comment:
Having assertions in the (Page Object side rather than Tests side) makes debugging defects more difficult. Let's refactor those Tests and Page Objects to include assertions on the tests side. So for example:
`Assert.assertTrue(manageForum.ForumExists()` etc.